src/client/components/ModalViewCancelEvent/ModalViewCancelEvent.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import { ModalCancelEvent } from './ModalViewCancelEvent';
+
+export default {
+  title: 'Modal Cancel Event',
+};
+
+export const CancelEventModal = () => <ModalCancelEvent />;
```

Comment:
You could use `action` to handle the callback here in Storybook, but it is not required.